command/read.go

```diff
@@ -37,6 +37,14 @@ Usage: vault read [options] PATH
 
       $ vault read secret/my-secret
 
+  Read entity details of a given ID:
```

Comment:
Hey @ncabatoff - any chance you can kindly commit these proposed textual changes? (I'm not sure if I completely get it).

Reply:
The bug you're trying to fix is https://github.com/hashicorp/vault/issues/16788, where the author complains about an example being given that might not apply in any given cluster, depending on whether `secret` is mounted or not.   So if you're leaving around examples that rely on `secret` being mounted as a kv engine, are you fixing the bug?

Reply:
@ncabatoff the concern of the requestor is to obtain a working example as what's provided at present is contingent on `secret` mount being present (irrespective of it being assumed to be KV or otherwise). Cluster consideration are not applicable here. In any case I'll update the `read` example to use `vault read auth/token/lookup-self`